src/components/data-management/ProjectDetails.jsx

```diff
@@ -232,17 +234,24 @@ const ProjectDetails = ({ width, height }) => {
       return;
     }
 
-    const statuses = ['uploaded', 'uploading', 'uploadError', 'fileNotFound'];
-    const newData = projects[activeProjectUuid]?.samples.map((sampleUuid, idx) => ({
-      key: idx,
-      name: samples[sampleUuid].name,
-      uuid: sampleUuid,
-      barcodes: _.sample(statuses),
-      genes: _.sample(statuses),
-      matrix: _.sample(statuses),
-      species: 'dataMissing',
-    }));
-
+    const newData = projects[activeProjectUuid].samples.map((sampleUuid, idx) => {
+      const sampleFiles = samples[sampleUuid].files;
+
+      const barcodesStatus = sampleFiles['barcodes.tsv.gz']?.status;
+      // TODO:  File name for genes can be one of these, should we rename to have only one possible name before putting it in S3?
+      const genesStatus = (sampleFiles['genes.tsv.gz'] ?? sampleFiles['features.tsv.gz'])?.status;
```

Comment:
It's ok to leave it. It's already handled in data-ingest: https://github.com/biomage-ltd/data-ingest/blob/cc303b3e2b942ff0b0e323c04671c98ffb17968b/src/1_Preproc.r#L35-L39